### ISSUE TITLE
Feature request: Skip 'Choose filter' diolog when only 1 filter

### ISSUE THREAD

**Issue by jeroenpraat** [daverix/urlforwarder#5]
If there is only one filter configured, is it an idea to skip the 'Choose filter' dialog? So that an user goes directly to the webbrowser.

**Comment by jeroenpraat** [comment]
Is this app abandoned?

**Comment by daverix** [comment]
Well... I'm working on it when I feel like it. Right now I have other stuff I prioritize more.

**Comment by jeroenpraat** [comment]
Thank you for your reply @daverix. If you _have_ time, maybe can give an answer on my question here: https://github.com/daverix/urlforwarder/issues/4

**Comment by daverix** [comment]
Well that would only work if you have one filter, right?

**Comment by jeroenpraat** [comment]
Yes that's right. But that's about this particular feature request, but I meant [my other issue](https://github.com/daverix/urlforwarder/issues/4) (has a higher priority for me).

**Comment by hansdez** [comment]
I was about to add this as a feature request, but then saw that somebody had mentioned this before. I only use Url forwarder for one thing too and this would save me a tap every time I use it. I guess the idea would be to no show the picker dialog when there is only only url filter defined but just pick that one. Would definitely be willing to buy you a beer/glass of decent wine/cup of coffee for that one…
